.changeset/change-to-typeerror.md

```
@@ -0,0 +1,41 @@
+
+```diff
+- } catch (error) {
+-   if (error instanceof RangeError) {
++ } catch (error) {
++   if (error instanceof TypeError) {
+      // Handle invalid content type or encoding
+    }
+  }
+```
+
+**New feature: Experimental compression format support**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)